code/modules/mob/dead/observer/observer.dm

```diff
@@ -148,6 +149,8 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 	show_data_huds()
 	data_huds_on = 1
 
+	creation_time = world.time
```

Comment:
```suggestion
	 COOLDOWN_START(src, creation_time , (CONFIG_GET(number/cooldown_antag_time) MINUTES))
```

Reply:
i still don't quite like the CONFIG_GET here maybe you could instead make it a static variable on the observer and grab it only once with an if instead but except for that i think this should be a better solution.

Reply:
now this should not be an issue anymore